### ISSUE TITLE
composer install fails

### ISSUE THREAD

**Issue by danielbachhuber** [WP-API/client-cli#6]
```
salty-wordpress ➜  wp-api-cli-client  composer install
Loading composer repositories with package information
Installing dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package requestsphp/oauth1 could not be found in any version, there may be a typo in the package name.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion> for more details.

Read <http://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

**Comment by danielbachhuber** [comment]
I assume you meant to include https://github.com/WP-API/OAuth1, but that has no composer.json nor is listed in packagist.org.

**Comment by rmccue** [comment]
Fixed back in 021e69c1f8acc5ffeeb9bfe6bb64fe54469b7e5f
